src/AutoRemoteSyscalls.cc

```diff
@@ -255,6 +256,16 @@ long AutoRemoteSyscalls::syscall_base(int syscallno, Registers& callregs) {
         // We entered the syscall, so stop now
         break;
       }
+      if (t->ptrace_event() == PTRACE_EVENT_EXIT) {
+        // We died, just let it be
```

Comment:
Is this a theoretical concern or did you actually see it happen?

Reply:
I saw it happen, but only due to an unrelated bug that caused the tracee to segfault on the first instruction out of a clone taking down the process before rr was done doing its setup. It's not aarch64 specific.